lib/archethic/mining/fee.ex

```diff
@@ -72,20 +72,16 @@ defmodule Archethic.Mining.Fee do
        ) do
     with {:ok, json} <- Jason.decode(content),
          "non-fungible" <- Map.get(json, "type", "fungible"),
-         utxos = [_ | _] <- Map.get(json, "properties", []) do
+         utxos when is_list(utxos) <- Map.get(json, "collection") do
```

Comment:
The former change was to ensure the list is not empty without iterating over the entire to know the size, maybe we should keep it to prevent empty NFT collection.

Reply:
ok